packages/treeprint/treeprint.2.3.0/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "2.0"
+synopsis: "Printing combinator library with automatic parenthese"
+description: """\
+Treeprint is a small pretty printing combinator library based on Format,
+with automatic parenthese ('(' and ')') insertion: building printing objects
+with their associativity (left/right/nonassoc) and connectivity precedences,
+objects are pretty-printed with parenthese when necessary."""
+maintainer: "jun.furuse@gmail.com"
+authors: "Jun Furuse"
+license: "MIT"
+homepage: "https://gitlab.com/camlspotter/treeprint"
+bug-reports: "https://gitlab.com/camlspotter/treeprint/-/issues"
+depends: [
+  "dune" {build & >= "2.0"}
```

Comment:
```suggestion
  "dune" {>= "2.0"}
```
Dune should not be tagged `build` for compatibility reasons (see https://github.com/ocaml/opam-repository/issues/21414)